Self-hosting build fail

### Component

Other

### Description

### Summary

Running `make bootstrap` fails with error: `No such built-in module: node:sqlite`

### Steps to Reproduce

-> Clone repo
-> setup .env
-> `cd server/ && make bootstrap`

### Expected Behavior

Docker containers in the compose file should spawn. 

### Actual Behavior
Snippet of the `make bootstrap` command's error log while building `server/dashboard/Dockerfile`:
```
0.7s
 => => extracting sha256:13e45b12880fbbfe3554ecf6b70131ace701d0c2fd3e2fb9abb842ba2494cd40                                                                         0.1s
 => => extracting sha256:cd322d0ddd02673e6c24a2158d1f12f6ec7c6fd9c0dc67123244d0f0eb0f8806                                                                         0.0s
 => [base 2/3] RUN apk add --no-cache libc6-compat                                                                                                                1.7s
 => [base 3/3] WORKDIR /app                                                                                                                                       0.0s
 => [deps 1/2] COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./                                                                                 0.0s
 => [builder 1/4] WORKDIR /app                                                                                                                                    0.0s
 => [runner 2/7] RUN addgroup --system --gid 1001 nodejs &&     adduser --system --uid 1001 nextjs                                                                0.3s
 => ERROR [deps 2/2] RUN   if [ -f yarn.lock ]; then yarn --frozen-lockfile --network-timeout 600000;   elif [ -f package-lock.json ]; then npm ci;   elif [ -f   1.9s
------
 > [deps 2/2] RUN   if [ -f yarn.lock ]; then yarn --frozen-lockfile --network-timeout 600000;   elif [ -f package-lock.json ]; then npm ci;   elif [ -f pnpm-lock.yaml ]; then corepack enable pnpm && pnpm i;   else npm install;   fi:
#0 0.914 ! Corepack is about to download https://registry.npmjs.org/pnpm/-/pnpm-11.4.0.tgz
#0 1.527 warn: This version of pnpm requires at least Node.js v22.13
#0 1.527 warn: The current version of Node.js is v20.20.2
#0 1.527 warn: Visit https://r.pnpm.io/comp to see the list of past pnpm versions with respective Node.js version support.
#0 1.733 node:internal/modules/cjs/loader:1031
#0 1.733       throw new ERR_UNKNOWN_BUILTIN_MODULE(request);
#0 1.733             ^
#0 1.733
#0 1.733 Error [ERR_UNKNOWN_BUILTIN_MODULE]: No such built-in module: node:sqlite
#0 1.733     at Module._load (node:internal/modules/cjs/loader:1031:13)
#0 1.733     at Module.require (node:internal/modules/cjs/loader:1289:19)
#0 1.733     at require (node:internal/modules/helpers:182:18)
#0 1.733     at ../store/index/lib/index.js (file:///root/.cache/node/corepack/v1/pnpm/11.4.0/dist/pnpm.mjs:24723:25)
#0 1.733     at __init (file:///root/.cache/node/corepack/v1/pnpm/11.4.0/dist/pnpm.mjs:15:56)
#0 1.733     at ../resolving/npm-resolver/lib/index.js (file:///root/.cache/node/corepack/v1/pnpm/11.4.0/dist/pnpm.mjs:36261:5)
#0 1.733     at __init (file:///root/.cache/node/corepack/v1/pnpm/11.4.0/dist/pnpm.mjs:15:56)
#0 1.733     at ../workspace/projects-graph/lib/index.js (file:///root/.cache/node/corepack/v1/pnpm/11.4.0/dist/pnpm.mjs:36399:5)
#0 1.733     at __init (file:///root/.cache/node/corepack/v1/pnpm/11.4.0/dist/pnpm.mjs:15:56)
#0 1.733     at ../workspace/projects-filter/lib/index.js (file:///root/.cache/node/corepack/v1/pnpm/11.4.0/dist/pnpm.mjs:43805:5) {
#0 1.733   code: 'ERR_UNKNOWN_BUILTIN_MODULE'
#0 1.733 }
#0 1.733
#0 1.733 Node.js v20.20.2
------
failed to solve: process "/bin/sh -c if [ -f yarn.lock ]; then yarn --frozen-lockfile --network-timeout 600000;   elif [ -f package-lock.json ]; then npm ci;   elif [ -f pnpm-lock.yaml ]; then corepack enable pnpm && pnpm i;   else npm install;   fi" did not complete successfully: exit code: 1
make: *** [up] Error 17
```

### Environment

- OS: macOS 15.6

--- a/server/dashboard/package.json
+++ b/server/dashboard/package.json
@@ -2,6 +2,7 @@
   "name": "mem0-dashboard",
   "version": "0.1.0",
   "private": true,
+  "packageManager": "pnpm@10.34.2",
   "scripts": {
     "dev": "next dev",
     "build": "next build",

